test/integration/conftest.py

```diff
@@ -171,16 +176,28 @@ def conservator(empty_db):
     else:
         admin_email = "admin@example.com"
 
-    empty_db.users.insert_one(
-        {
-            "_id": Conservator.generate_id(),
-            "role": ADMIN_ROLE,
-            "name": "admin user",
-            "email": admin_email,
-            "apiKey": api_key,
-            "organizationId": organization["_id"],
-        }
-    )
+    admin_found = False
```

Comment:
This variable seems redundant to me. The `if` statement below could be rewritten as:
```
if user: 
...
else:
    # create user
```

Reply:
Done.